src/test/lib/operators/operator_join_predicate_test.cpp

```diff
@@ -49,4 +49,21 @@ TEST_F(OperatorJoinPredicateTest, FromExpressionImpossible) {
   ASSERT_FALSE(predicate_b);
 }
 
+TEST_F(OperatorJoinPredicateTest, Flip) {
+  auto predicate = OperatorJoinPredicate{{ColumnID{0}, ColumnID{1}}, PredicateCondition::LessThanEquals};
+  EXPECT_FALSE(predicate.is_flipped());
+
+  predicate.flip();
+  EXPECT_TRUE(predicate.is_flipped());
+  EXPECT_EQ(predicate.column_ids.first, ColumnID{1});
+  EXPECT_EQ(predicate.column_ids.second, ColumnID{0});
+  EXPECT_EQ(predicate.predicate_condition, PredicateCondition::GreaterThanEquals);
```

Comment:
Maybe the flip should be tested for some more PredicateConditions.

Reply:
Not sure if that fits here. Flipping the predicate conditions is implemented in `types.hpp/.cpp` and maybe we could tests for the methods defined there. However, the tests would be pretty much 1:1 reimplementation, as we do simple mappings there.